examples/liqpool/src/lib.rs

```diff
@@ -328,7 +328,7 @@ mod test {
     use stellar_contract_sdk::testing::mem::{MemHost, MemLedgerKey, MemLedgerVal, MemObj};
     use stellar_contract_sdk::testing::{swap_mock_host, with_mock_host};
     use stellar_contract_sdk::Object;
-    use stellar_strkey::PublicKeyEd25519;
+    use stellar_strkey::PublicKey as StrkeyPublicKey;
```

Comment:
@tomerweller Since the strkey package's PublicKey will always collide with the xdr packages PublicKey, I wonder if we should just name it `StrkeyPublicKey` in the strkey crate 🤔. It's more likely to result in us naming it the same thing everywhere. Wdyt?